src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/config/HmppsAssessForEarlyReleaseApiExceptionHandler.kt

```diff
@@ -134,6 +123,17 @@ class HmppsAssessForEarlyReleaseApiExceptionHandler {
     return problemDetail
   }
 
+  @ExceptionHandler(Exception::class)
```

Comment:
does order matter technically? I thought it did but regardless it makes sense to have this last

Reply:
I think it's base on the the exception hierarchy,... derived exception first and then down the parent tree. But I think having it here makes sense as it's the parent to most.